src/docs/admin/references/NodeSourceInfrastructureReference.adoc

```diff
@@ -212,7 +212,7 @@ now be added using the *Create Node Source* panel in the Resource Manager or the
 
 As AWS is a paying service, when the Scheduler is stopped normally (without removing the created infrastructure), all the created AWS instances will be terminated. And when the Scheduler is restarted, these instances will be re-configured as per previous settings.
 
-WARNING: If the Scheduler is forced killed, the created AWS instances will not be terminated. And, when the Scheduler is restarted, the infrastructure will be re-configured as per previous settings. If the instances were delete at the AWS side, they will be re-created and re-configured.
+WARNING: If the ProActive server is forced killed, the created AWS instances will not be terminated. And, when the ProActive server is restarted, the infrastructure will be re-configured as per previous settings. If the instances were delete at the AWS side, they will be re-created and re-configured.
```

Comment:
- ProActive server  is **forcibly killed**
- be re-configured as per **the** previous settings.
- If the instances were **deleted** at

Reply:
A small point to confirm, "be re-configured as **per** the previous settings." should be "be re-configured as the previous settings.", right?

Reply:
To me 'be re-configured as per the previous settings' is better. I'll let you choose the proper sentence according to you.